MovieDB/Platform/Repositoties/FavoriteRepository.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  FavoriteRepository.swift
+//  MovieDB
+//
+//  Created by nguyen.van.hungd on 6/10/19.
+//  Copyright © 2019 nguyen.van.hungd. All rights reserved.
+//
+
+import Realm
+import RealmSwift
+
+protocol FavoriteRepositoryType {
+    func getListFavorite() -> [Movie]
+    func deleteMovie(id: Int)
+}
+
+final class FavoriteRepository: FavoriteRepositoryType {
+    func getListFavorite() -> [Movie] {
+        do {
+            let movies = try Realm().objects(MovieEntity.self)
+            return movies.map({
+                var model = Movie()
```

Comment:
```suggestion
                var model = Movie().with {}
```
sử dụng lib Then cho gọn nhé

Reply:
Chỗ này movies.map đang giữ $0 rồi nên nếu dùng with{} sinh ra thêm 1 $0 nữa